chainerx_cc/chainerx/CMakeLists.txt

```diff
@@ -128,55 +128,38 @@ target_link_libraries(
     absl::flat_hash_map
     )
 
-if(MSVC)
-    target_link_libraries(chainerx
-        PUBLIC
-        ${CMAKE_DL_LIBS})
-
-    target_link_options(chainerx PUBLIC /wholearchive:$<TARGET_FILE:chainerx_routines>)
-    target_link_options(chainerx PUBLIC /wholearchive:$<TARGET_FILE:chainerx_native>)
-    target_link_options(chainerx PUBLIC /wholearchive:$<TARGET_FILE:chainerx_testing>)
-    target_link_libraries(chainerx PRIVATE chainerx_routines)
```

Comment:
Here chainerx_base was not linked, but on the new changeset, it is.
I am not sure if this might cause any kind of issue in windows.
I don't think it should at all but might be better to confirm it?

Reply:
Fixed in afacf6919f94ee013642b00d8a1e9604600ace65.
Unfortunately I don't have Windows environment to test it.
Although it's better to confirm the fix, we don't necessarily have to do it because Windows is currently not officially supported.

Reply:
That sounds perfectly fine to me.
Thanks for addressing it
LGTM then!